app/views/maps/_home_map_filters.html.erb

```diff
@@ -50,8 +50,13 @@
         </fieldset>
       </div>
 
-      <div class="margin-bottom-2">
-        <h4 class="font-sans-md bold-height-2 margin-bottom-105 margin-top-0">Innovation status</h4>
+      <div class="margin-bottom-3">
+        <h4 class="font-sans-md bold-height-2 margin-bottom-1 margin-top-0">
+          <div href="#facility-status-modal" class="cursor-pointer" aria-controls="facility-status-modal" data-open-modal>
+            <span class="usa-sr-only">Open facility status modal</span>
+            <span>Innovation <u class="dotted">status</u></span>
+          </div>
+          <span><%= render partial: '/va_facilities/facility_status_definitions_modal', locals: {is_map_filter: true} %></span></h4>
```

Comment:
I don't think we need the `<span>` around this partial and the partial can be outside of the `<h4>`.

Reply:
Got it.